.github/workflows/focus-ios-smoke

```diff
@@ -0,0 +1,121 @@
+name: "Focus Smoke Tests"
+    
+on:
+    workflow_dispatch:
+
+env:
+    browser: focus-ios
+    xcode_version: 15.4
+    ios_version: 17.5
+    ios_simulator_default: iPhone 15
+    xcodebuild_test_plan: SmokeTest
+    xcodebuild_scheme: Focus
+    xcodebuild_target: XCUITest
+    test_results_directory: /Users/runner/tmp
+    
+jobs:
+    compile:
+        name: Compile
+        runs-on: macos-14-large
+        steps:
+            - name: Check out source code
+              uses: actions/checkout@v4.1.7
+            - name: Setup Xcode
+              id: xcode
+              run: |
+                sudo rm -rf /Applications/Xcode.app
+                sudo xcode-select -s /Applications/Xcode_${{ env.xcode_version }}.app/Contents/Developer
+                xcodebuild -version
+                ./checkout.sh
+                ./bootstrap.sh --force
+            - name: Compile source code
+              id: compile
+              run: |
+                xcodebuild \
+                  -resolvePackageDependencies \
+                  -onlyUsePackageVersionsFromResolvedFile
+                xcodebuild \
+                  build-for-testing \
+                  -scheme ${{ env.xcodebuild_scheme }} \
+                  -target ${{ env.xcodebuild_target }} \
+                  -derivedDataPath ~/DerivedData \
+                  -destination 'platform=iOS Simulator,name=${{ env.ios_simulator_default }},OS=${{ env.ios_version }}'
+              working-directory: ${{ env.browser }}
+            - name: Save Derived Data
+              id: upload-derived-data
+              uses: actions/upload-artifact@v4.3.3
+              with:
+                name: xcode-cache-deriveddata-${{ github.workflow }}-${{ github.sha }}
+                path: ~/DerivedData/Build/Products
+                retention-days: 2          
+    run-tests:
+        name: Run tests
+        runs-on: macos-14-large
+        needs: compile
+        strategy:
+            fail-fast: false
+            matrix:
```

Comment:
I can use a matrix to run both full functional tests and smoke tests. We can have the following permutations:
* Full Functional Tests on iPhone 15
* Full Functional Tests on iPad Pro
* Smoke Tests on iPhone 15
* Smoke Tests on iPad Pro